src/css/controls/imports/playback-label.less

```diff
@@ -15,17 +15,16 @@
 .jw-idle-label {
     border-radius: 50%;
     color: @white;
-    filter: drop-shadow(1px 1px 5px rgba(12, 26, 71, 0.25));
     font: normal 16px/1 Arial, Helvetica, sans-serif;
     position: relative;
-    transition: background-color @default-transition;
-    transition-property: background-color, filter;
```

Comment:
So we saw it transitioning the `transform` of the element, not the `background-color`, `filter`. Any idea why removing these properties prevents the element from rotating?